src/index.ts

```diff
@@ -0,0 +1,24 @@
+console.log('Try npm run lint/fix!');
```

Comment:
We might want to remove this. There probably shouldn't be a src dir at the root of the monorepo especially if it's just for testing stuff.

Reply:
I'll take care of this and reach out before any merges are done!